tidb-server/main.go

```diff
@@ -70,6 +70,7 @@ var (
 	slowThreshold       = flag.Int("slow-threshold", 300, "Queries with execution time greater than this value will be logged. (Milliseconds)")
 	queryLogMaxlen      = flag.Int("query-log-max-len", 2048, "Maximum query length recorded in log")
 	tcpKeepAlive        = flag.Bool("tcp-keep-alive", false, "set keep alive option for tcp connection.")
+  proxyProtocol       = flag.String("proxy-protocol", "", "proxy protocol allowed IP or *, empty mean do not use proxy protocol")
```

Comment:
I think the name of "proxy-protocol" is confusing. In fact, this option does not set the "protocol", instead it sets the allowed proxy address for this protocol. Maybe you can rename to "proxy_protocol_networks" like [percona](https://www.percona.com/doc/percona-server/LATEST/flexibility/proxy_protocol_support.html).